internal/setup/setup.go

```diff
@@ -71,11 +71,14 @@ func Setup(ctx context.Context, config DatabaseConfigProvider) (*serverenv.Serve
 	var sm secrets.SecretManager
 	if provider, ok := config.(SecretManagerConfigProvider); ok {
 		smConfig := provider.SecretManagerConfig()
+		if err := envconfig.Process(ctx, smConfig, nil); err != nil {
```

Comment:
Do we need this here? It's down on line 91. Won't that re-process?

Reply:
we need to pre-process the secret manager config, so the default (or overridden) secret manager can be used to process the rest of the configs.

If you try to use a secret:// to configure secret manager, you've broken the space time continuum.

Reply:
https://github.com/google/exposure-notifications-server/pull/500